common/model/opening-hours.js

```diff
@@ -10,22 +10,37 @@ export type OpeningHoursDay = {|
   note?: string
 |};
 
-type ExceptionalOpeningHoursDay = {|
+export type ExceptionalOpeningHoursDay = {|
   ...OpeningHoursDay,
-  overrideDate: Date
+  overrideDate: Date,
+|}
+
+export type SpecialOpeningHours = {|
+  opens: string,
+  closes: string,
+  validFrom: Date,
+  validThrough: Date
 |}
 
 export type OpeningHours = {|
   regular: OpeningHoursDay[],
-  exceptional?: ExceptionalOpeningHoursDay[]
+  exceptional?: ?ExceptionalOpeningHoursDay[]
 |};
 
-type Place = {|
+export type Place = {|
```

Comment:
We have another type of Place.

Let's beware, especially as they're both exported:
https://github.com/wellcometrust/wellcomecollection.org/blob/master/common/model/place.js